### DISCUSSION TITLE
Add 'since version' for hooks

### DISCUSSION THREAD

**Discussion by ar7casper** [mantinedev/mantine#6440] [Feature requests]
It could be awesome of the hooks pages had the mantine version they were added in.
I had an issue now where a hook I wanted to use doesn't exist in my current version.
Initially I thought I had a bug in my code, wasted some time :)

**Comment by rtivital** [comment]
Not planned, the documentation is always up-to-date for the latest version. It is usually better to update to the latest version anyway as it includes bug fixes and new features, minor versions do not include breaking changes.
